staging/src/k8s.io/apimachinery/pkg/labels/labels.go

```diff
@@ -29,8 +29,8 @@ type Labels interface {
 	// Has returns whether the provided label exists.
 	Has(label string) (exists bool)
 
-	// Get returns the value for the provided label.
-	Get(label string) (value string)
+	// Get returns the value and whether exists for the provided label.
+	Get(label string) (value string, exists bool)
```

Comment:
this is breaking change to the interface, so better to check with kubernetes-sig-api-machinery@googlegroups.com on whether it is acceptable.

Reply:
Thanks for your suggestion.